### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by boldjoel**
You will need to compile your own processpaf

### CODE REVIEW

**Comment by lgarithm** [comment]
Let's remove the whole pafprocess folder if we are not going to modify it.

File: inference/tensblur/1k_read.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+val_path = '/Users/Joel/Desktop/1k_list.txt'
```
Comment (lgarithm) [inline inference/tensblur/1k_read.py]: Is this file needed?

Comment (zsdonghao) [reply inference/tensblur/1k_read.py]: It is useless, Joel say
